magenta/main.py

```
@@ -0,0 +1,524 @@
+    # Music VAE 모델 체크포인트 다운로드 (필요한 경우)
+    if not os.path.exists(MUSIC_VAE_CHECKPOINT):
+        try:
+            print(f"Music VAE 모델 체크포인트 다운로드 중... ({MUSIC_VAE_MODEL_URL})")
+            response = requests.get(MUSIC_VAE_MODEL_URL, stream=True)
+            response.raise_for_status()
+            
+            with open(MUSIC_VAE_CHECKPOINT, 'wb') as f:
+                for chunk in response.iter_content(chunk_size=8192):
+                    f.write(chunk)
+            
+            print(f"Music VAE 모델 체크포인트가 다운로드되었습니다.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)